cmake/onnxruntime_providers.cmake

```diff
@@ -502,7 +502,7 @@ if (onnxruntime_USE_CUDA)
 
   install(TARGETS onnxruntime_providers_cuda
           ARCHIVE  DESTINATION ${CMAKE_INSTALL_LIBDIR}
-          LIBRARY  DESTINATION ${CMAKE_INSTALL_LIBDIR}
+          LIBRARY  DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
Does it mean on Linux it will be installed to /usr/local/bin instead of /usr/local/lib ?

Reply:
Yeah, I can add some code, wait for a while.